interface/stylesheet.dm

```diff
@@ -135,6 +135,7 @@ h1.alert, h2.alert		{color: #000000;}
 .noticealien			{color: #00c000;}
 .alertalien				{color: #00c000;	font-weight: bold;}
 .changeling				{color: #800080;	font-style: italic;}
+.assimilator			{color: #800080;	font-size: 2 ; font-weight: bold;}
```

Comment:
This needs to be in browserOutput.css this is just a fallback.

Reply:
It's in there as well